ZukoAzula2016/src/org/team1540/zukoazula/Portcullis.java

```diff
@@ -21,22 +21,28 @@
     private static final FloatIO leftEncoder = leftGrabMotor.modEncoder().getEncoderPosition();
     private static final FloatIO rightEncoder = rightGrabMotor.modEncoder().getEncoderPosition();
 
-    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.3f);
+    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.3f).negated();
```

Comment:
Oh, nice! Now I don't have to do that on my version of the code and worry about reversing all of the motors and PID and stuff. :+1: